### ISSUE TITLE
Use Progressive Image to display avatar in AccountDetails.

### ISSUE THREAD

**Issue by jainkuniya** [zulip/zulip-mobile#3352]
Currently we are showing image with `getMediumAvatar`  as url. But on slow internet connections it takes time to load image and in this time nothing is shown, empty screen. Which doesn't look good. So till image is completely loaded, show a thumbnail image i.e the avatar which we usually show in message list. Also add a smooth transition between real image and the thumbnail image, like https://medium.com/react-native-training/progressive-image-loading-in-react-native-e7a01827feb7

@zulipbot claim

**Comment by jainkuniya** [comment]
Demo can be tested here: https://github.com/jainkuniya/zulip-mobile/tree/issue-3352

**Comment by borisyankov** [comment]
Related but separate issue: we should be showing a higher resolution avatar in `AccountDetails`.
Would you look into that too?

**Comment by jainkuniya** [comment]
By `Higher resolution` you mean the url with `-medium.png` ?

**Comment by borisyankov** [comment]
Yeah.  I am not sure if we should be using `medium` there or at all.
I don't remember the details, worked on these too long ago.
Also, it is totally possible that the original decision was not correct :)
All I know is that often when looking at the `AccountDetailsScreen` the avatar looks blurry from the lower-than-ideal resolution we are using.

**Comment by jainkuniya** [comment]
> All I know is that often when looking at the `AccountDetailsScreen` the avatar looks blurry from the lower-than-ideal resolution we are using.

#3359 is handling this :)
